### ISSUE TITLE
[a11y] Can't navigate between different ChoiceGroups with arrow keys

### ISSUE THREAD

**Issue by sehartne** [microsoft/fluentui#21398]
### Environment Information

- **Browser**: Edge Version 96.0.1054.2 (Official build) dev (64-bit)
- **OS**: Windows 11 OS Build 22458.1000 

### Describe the issue:

When there are multiple different ChoiceGroups in a sequence the arrow keys can't be used to navigate between the radio buttons of different ChoiceGroups, they just loop in the radio buttons of the ChoiceGroup that first got focus. The layout of these ChoiceGroups also can't have their layout changed from vertical. There should be a way to connect different ChoiceGroups as connecting two separate radio groups are supported in standard HTML.

### Please provide a reproduction of the issue in a codepen:

1. Open Codepen link: https://codepen.io/sharto12/pen/podzaLy
2. Focus to the first choice group
3. Use arrow keys to navigate through options
4. Can't move on to second choice group with out tabbing

#### Actual behavior:

Multiple ChoiceGroups cannot be linked allowing the keyboard navigation of multiple groups

#### Expected behavior:

Keyboard navigation across multiple ChoiceGroups

**Comment by TristanWatanabe** [comment]
This behavior is achieved by passing the same `name` attribute to the multiple `ChoiceGroups` along with passing the same `selectedKey` to them to put the `ChoiceGroups` in controlled mode. Codepen example: https://codepen.io/tristanwatanabe/pen/VwrpPJJ
